config/env.sample

```diff
@@ -0,0 +1,5 @@
+PORT=5005
+CERT_FILE="config/public-cert.pem"
+KEY_FILE="config/private-key.pem"
+KEY_PASS=KEY_FILE
```

Comment:
whoops, meant for that to be `KEY_PASS`. Are you suggesting we leave out all keys except for `AUTH_TESSEL_OA2`?